ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.ts

```diff
@@ -402,6 +401,36 @@ const triggerSet: TriggerSet<Data> = {
       netRegexKo: NetRegexes.startsUsing({ id: '5C7[34]', source: ['헨젤', '그레텔'], capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'Paradigm Hansel/Gretel Bloody Sweep',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5C5[4567]', source: ['Hansel', 'Gretel'] }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
```

Comment:
Sorry, I didn't see this before, but is there a reason to delay this?

Reply:
Not particularly other than the alert starting super early relative to the actual ability cast (8 seconds or 12 seconds) which means the alert will have disappeared a long time before. But.. I guess that's only more important if you need to do something precisely on time... now that I think about it just moving ahead of time is fine, so I'll go ahead and remove this.